### ISSUE TITLE
/dev/input: No such file or directory

### ISSUE THREAD

**Issue by eraserix** [fyne-io/fyne#483]
**Describe the bug**

When starting the fyne_demo application, I get the following error:

```
GLFW: An uncaught error has occurred: PlatformError: Linux: Failed to watch for joystick connections in /dev/input: No such file or directory
GLFW: Please report this bug in the Go package immediately.
2019/10/09 11:28:30 PlatformError: Linux: Failed to open joystick device directory /dev/input: No such file or directory
```

I indeed do not have /dev/input inside the container, because I did not forward it any of the devices into the container. I don't see a problem with not having this directory.

**Device (please complete the following information):**
 - OS: Linux Debian 8 Jessie in a LXD container
 - Go version: go version go1.12.4 linux/amd64
 - Fyne version: 1180436f60e895bde9e4e38fa2f8aacecf6ba1f4

**Comment by andydotxyz** [comment]
Without /dev/input how are user events forwarded to your container?

**Comment by eraserix** [comment]
I don't know, but various GUIs seem to do fine without /dev/input (also fyne-GUIs, btw).

I use ssh (but without X-Forwarding) to log into the container and have a bindmount in place for the X11-socket. Maybe the events are received from X11?

**Comment by andydotxyz** [comment]
Oh, hold on - are you saying this is a reported warning but that the application works as expected?

**Comment by andydotxyz** [comment]
Possibly fixed by #183 once we get 3.3 from upstream

**Comment by eraserix** [comment]
Yes, the fyne-demo works OK, it just prints this error during startup. Sorry for not having been more clear on that.

**Comment by andydotxyz** [comment]
OK, this will get fixed when we update to 3.3 which is held up on an external project.
In the meantime as this is just a warning I'll close the issue, I hope you don't mind - there is nothing we can do to progress this
